File: BOJ/Java/단계별로_풀어보기/B_03_반복문/BOJ_11021.java
package 단계별로_풀어보기.B_03_반복문;

/**
 * BOJ 11021 : A+B - 7
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class BOJ_11021 {

	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		StringBuilder sb = new StringBuilder();

		while (true) {

			String str = br.readLine();

			int A = str.charAt(0) - 48;
			int B = str.charAt(2) - 48;

			if (A == 0 && B == 0) {
				break;
			}
			sb.append((A + B)).append('\n');
		}
		System.out.println(sb);
	}
}

File: BOJ/CPP/단계별로_풀어보기/03_반복문/BOJ_11021.cpp
/*
	BOJ 11021 : A+B - 7

	author : kdgyun
	link https://st-lab.tistory.com
	link https://github.com/kdgyun
	Created by kdgyun
	Copyright © 2021 kdgyun. All rights reserved.
*/

#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {

	ios_base::sync_with_stdio(false);	// 두 표준 입출력 동기화 해제
	cin.tie(NULL);	// 입력과 출력이 묶여있는 것을 풀어준다.

	int T;
	cin >> T; // 테스트 케이스

	for (int i = 1; i <= T; i++) {
		int a;
		int b;
		cin >> a >> b;

		cout << "Case #" << i << ": " << a + b << "\n"; 
	}
	return 0;
}
